drivers/TinyGSM/TinyGsmFifo.h

```diff
@@ -0,0 +1,153 @@
+#ifndef TinyGsmFifo_h
+#define TinyGsmFifo_h
+template <class T, unsigned N>
+class TinyGsmFifo
```

Comment:
I believe the suppression need to be placed here, just before the class declaration.

Reply:
See here for reference: https://github.com/mysensors/MySensors/blob/b5a2fc226fe64759f2cb47a255b9b6f34850a968/drivers/ATSHA204/sha256.cpp#L36